strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -50,33 +49,21 @@ class Circuit:
          The state of the modes is manipulated by calling the various methods."""
 
     # pylint: disable=too-many-instance-attributes,too-many-public-methods
-    def __init__(self, graph, num_modes, cutoff_dim, pure, batch_size):
-        self._graph = (
-            None  # will be set when reset is called below, but reset needs something to compare to
-        )
+    def __init__(self, num_modes, cutoff_dim, pure, batch_size):
         self._hbar = 2
         self.reset(
-            graph=graph,
-            num_subsystems=num_modes,
-            pure=pure,
-            cutoff_dim=cutoff_dim,
-            batch_size=batch_size,
+            num_subsystems=num_modes, pure=pure, cutoff_dim=cutoff_dim, batch_size=batch_size
         )
 
     def _make_vac_states(self, cutoff_dim):
         """Make vacuum state tensors for the underlying graph"""
-        with self._graph.as_default():
```

Comment:
Just curious for my own understanding: how come it was decided to remove having the graph stored?

Reply:
graphs are no longer used by default in tensorflow2.0

Reply:
Oh got ya, thanks! :blush: